src/blocks/map/view.js

```diff
@@ -0,0 +1,248 @@
+// eslint-disable-next-line import/no-unresolved
+import mapboxgl from '!mapbox-gl';
```

Comment:
I actually don't know this syntax. What does the `!` mean here?

Reply:
Its from the documentation, and a new one on me too. I had a search around and found this: https://stackoverflow.com/questions/48845124/what-does-exclamation-point-mean-before-and-after-in-javascript it seems to override a part of the webpack loader.